cli/cli_app.go

```diff
@@ -377,7 +376,26 @@ func runTerraformCommandIfNoErrors(possibleErrors error, terragruntOptions *opti
 		return err
 	}
 
-	return shell.RunTerraformCommand(terragruntOptions, terragruntOptions.TerraformCliArgs...)
+	var err error = nil
+
+	// Retry the command configurable time with sleep in between
```

Comment:
Could you extract this into a method named something like `runTerraformCommandWithRetries`?

Reply:
Done